README.md

```diff
@@ -11,6 +11,12 @@ AppleScript for [iTerm2](https://iterm2.com/) [Alfred](https://www.alfredapp.com
 4. Select the text in the box.
 4. Paste.
 
+## Optional property settings:
+
+1. Set `open_in_new_window` to `true` to always open command in new window.
```

Comment:
Please make the numbers correct (I know it’s only aesthetic for the Markdown and that above they’re not, those should be fixed as well).